lib/awskeyring/awsapi.rb

```diff
@@ -110,8 +121,8 @@ def self.get_login_url(key:, secret:, token:, path:, user:) # rubocop:disable  M
                                    secret_access_key: secret)
 
         session = sts.get_federation_token(name: user,
```

Comment:
Just noticed that you might hit the same deal here with the name length being limited to 32 characters

Reply:
I'll get this merged and the 32 character check can be fixed afterwards.